### ISSUE TITLE
Stage 1 Training on UCF-Crime

### ISSUE THREAD

**Issue by borijang** [fjchange/MIST_VAD#35]
Hi, thanks for the amazing work. I would like to train MIST on UCF-Crime from scratch. As I see, the training code for stage 1 is for SHT. What needs to be modified in order for it to work on UCF?

**Comment by TianmingXie** [comment]
I have successfully debug the stage1training. But during the stage2training, I was told lack of the logger path. Can you told me how can I produce the log file?

**Comment by seetlin** [comment]
> I have successfully debug the stage1training. But during the stage2training, I was told lack of the logger path. Can you told me how can I produce the log file?

I don't know if it's a success,stage1 training accuracy rate is only 88 and not 90. Using the file generated by this first step to train the second stage, the cashback is a value from beginning to end.

**Comment by albertxwz** [comment]
> 

In the paper, it seems that auc of stage 1 for SHT_I3D is 89.15%. However, I cannot reproduce it using the set of parameters in the source code and I just got about 87%.

**Comment by albertxwz** [comment]
> > I have successfully debug the stage1training. But during the stage2training, I was told lack of the logger path. Can you told me how can I produce the log file?
> 
> I don't know if it's a success,stage1 training accuracy rate is only 88 and not 90. Using the file generated by this first step to train the second stage, the cashback is a value from beginning to end.

In the paper, it seems that auc of stage 1 for SHT_I3D is 89.15%. However, I cannot reproduce it using the set of parameters in the source code and I just got about 87%.

**Comment by F3NGYUX1N** [comment]
> Hi, thanks for the amazing work. I would like to train MIST on UCF-Crime from scratch. As I see, the training code for stage 1 is for SHT. What needs to be modified in order for it to work on UCF?
Hello, the stage1 part of SHT_I3D has been trained, but the code lacks MIL.ph and cannot save the false label. May I ask how you conducted the stage2 part? Looking forward to your reply.

**Comment by F3NGYUX1N** [comment]
> In the paper, it seems that auc of stage 1 for SHT_I3D is 89.15%. However, I cannot reproduce it using the set of parameters in the source code and I just got about 87%.

Hello, the stage1 part of SHT_I3D has been trained, but the code lacks MIL.ph and cannot save the false label. May I ask how you conducted the stage2 part? Looking forward to your reply.

**Comment by albertxwz** [comment]
> > In the paper, it seems that auc of stage 1 for SHT_I3D is 89.15%. However, I cannot reproduce it using the set of parameters in the source code and I just got about 87%.
> 
> Hello, the stage1 part of SHT_I3D has been trained, but the code lacks MIL.ph and cannot save the false label. May I ask how you conducted the stage2 part? Looking forward to your reply.

Thank you for your reply. I conducted your code based on [Training_Guidelines.md](https://github.com/fjchange/MIST_VAD/blob/master/Training_Guidelines.md).

**Comment by zzyshowers** [comment]
请问大家都怎么调试阶段一的训练，我只出来set seed0？
